source/kraken/kraken_zmq.h

```diff
@@ -161,8 +162,8 @@ inline void doWork(zmq::context_t& context,
 
         auto start_timestamp = (start - navitia::posix_epoch).total_milliseconds();
         auto end_timestamp = (end - navitia::posix_epoch).total_milliseconds();
-        LOG4CPLUS_INFO(logger, "Api : " << pbnavitia::API_Name(api) << ", worker : " << worker_id
-                                        << ", request : " << request_id << ", start : " << start_timestamp
-                                        << ", end : " << end_timestamp);
+        LOG4CPLUS_INFO(logger, "Api : " << pbnavitia::API_Name(api) << ", hostname: " << hostname
```

Comment:
```suggestion
        LOG4CPLUS_INFO(logger, "Api : " << pbnavitia::API_Name(api) << ", hostname: " << navitia::get_hostname();
```
How name is consistent across threads. Any specific reason to pass it as an argument ?

Reply:
i'm not sure the `get_hostname()` is thread safe...